docs/en/tutorials/getting-started.rst

```diff
@@ -274,9 +267,10 @@ language. The metadata language describes how entities, their
 properties and references should be persisted and what constraints
 should be applied to them.
 
```

Comment:
I couldn't find any justification throughout Doctrine's documentation that, "References in the text will be made to the XML mapping.". It seems like the default is the docblock style. Accordingly, I removed this line.